Makefile

```
@@ -301,9 +370,16 @@ dist: apply server webapp bundle
 dist-ci: apply server-ci webapp bundle
 
-## Builds and installs the plugin to a server.
+## Builds and installs the plugin to a server. Builds linux-amd64 only by
+## default (the platform of most Mattermost server deployments). Set
+## MAKE_ALL_PLATFORMS=1 to build all supported OS/arch combinations — needed
+## when deploying to a Mac- or Windows-native Mattermost.
 .PHONY: deploy
+ifeq ($(MAKE_ALL_PLATFORMS),)
+deploy: dist-ci
+else
 deploy: dist
```

checkmake (0.3.2)

[warning] Target "deploy" defined multiple times (lines 379 and 381).

(uniquetargets)